### PR TITLE
Edit README to clarify setup instructions where I ran into issues

### PR DESCRIPTION
**Pull request by Nateowami**
These changes to the README are mainly relating to issues I ran into while setting up a development environment. There are more issues I ran into, but those need some discussion first.

The one place I'm not sure I interpreted correctly is [line 167](https://github.com/sillsdev/web-languageforge/commit/f5480f5c71f4de7c09da4237fb6c7d097a964f15#diff-04c6e90faac2675aa89e2176d2eec7d8R167):
``` diff
- Modify `/usr/local/lib/node_modules/jscs/presets/airbnb.json` and set the line 
+ Modify `/usr/local/lib/node_modules/jscs/presets/airbnb.json` and change `"requireTrailingComma"` to
  `"disallowTrailingComma": true,`
```

### CODE REVIEW

**Comment by Nateowami** [comment]
Just added another commit clarifying
- git submodules - How to init them after a clone.
- What to do if/when Ansible crashes on the `ssl_config : LetsEncrypt: Install packages` task (run it again).

**Comment by darcywong00** [comment]
Yes, the airbnb.json file needs to change 
from `"requireTrailingComma": { "ignoreSingleLine": true },`
to `"disallowTrailingComma": true,`

File: README.md [hunk 1]
```diff
@@ -61,17 +61,18 @@ Otherwise just create a symbolic link between languageforge and scriptureforge..
 ln -s web-languageforge web-scriptureforge
 ```
 
-Change the variable *mongo_path: /var/lib/mongodb* in `deploy/vars_palaso.yml`
+Change the variable *mongo_path: /var/lib/mongodb* in `deploy/vars_palaso.yml`. Set it to a location where MongoDB should store its databases.
  - **Vagrant VM Setup**: uncomment line 6 and comment line 5
- - **Local Linux Development Setup**: uncomment line 5 and comment line 6 (or whatever is appropriate on your system, its best to have mongo on you HDD rather than SDD). 
+ - **Local Linux Development Setup**: uncomment line 5 and comment line 6 (or whatever is appropriate on your system, its best to have Mongo store databases on you HDD rather than SSD). Make sure the `mongodb` user has permission to read and write to the path you specify.
```
Comment (darcywong00) [inline README.md]: can you also fix this earlier typo? "...Mongo store database on you**r** HDD..."

Comment (Nateowami) [inline README.md]: Sure. I missed that one. Good catch.

Comment (Nateowami) [inline README.md]: Fixed.

**Comment by irahopkinson** [comment]
lgtm, assuming you incorporate Darcy's comment.

**Comment by darcywong00** [comment]
re: this morning's discussion, can you see about removing this line

```
zend_extension = /usr/lib/php/20151012/xdebug.so
```

in both php.in files?

**Comment by Nateowami** [comment]
I talked to Ira about the Xdebug configuration and have pushed a commit to update the instructions accordingly. On my machine Xdebug is running both on the CLI and on Apache without having `zend_extension = /usr/lib/php/20151012/xdebug.so` in either php.ini file.

The file `/etc/php/7.0/mods-available/xdebug.ini` is just `zend_extension=xdebug.so` and as I understand loads Xdebug for both Apache and the CLI.

I also removed the following config options:
``` ini
xdebug.remote_connect_back=1
xdebug.scream=0

```

**Comment by Nateowami** [comment]
Just pushed a commit clarifying what should be changed in airbnb.json

**Comment by irahopkinson** [comment]
lgtm

**Pull request merged**